backend/src/test/controller/BaseController.ts

```diff
@@ -1,13 +1,21 @@
 import express, { Express } from 'express';
 import { App } from '../../main/App';
+import redisMock, { RedisClient } from 'redis-mock';
 
 /**
  * Creates an express app and registers all handlers and routes
  * * Used for integration tests that test controllers (supertest needs an instance of express app without the port binding)
  * */
-export const registerTestApplication = (): Express => {
+export const registerTestApplication = async (
+  redisClient: RedisClient
+): Promise<Express> => {
   const app = new App();
   const expressApp = express();
-  app.registerHandlersAndRoutes(expressApp);
+  await app.registerHandlersAndRoutes(expressApp, redisClient);
   return expressApp;
 };
+
+export const connectRedisClient = async (): Promise<RedisClient> => {
+  // see https://www.npmjs.com/package/redis-mock
+  return redisMock.createClient();
```

Comment:
Oooh nice that you found a mock for it :)

Reply:
Eric actually found this one! It turned out to be really simple which is nice :)